p2p/module_test.go

```diff
@@ -0,0 +1,108 @@
+package p2p
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/pokt-network/pocket/runtime/configs"
+	"github.com/pokt-network/pocket/runtime/defaults"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_configureBootstrapNodes(t *testing.T) {
+	defaultBootstrapNodes := strings.Split(defaults.DefaultP2PBootstrapNodesCsv, ",")
+	type args struct {
+		p2pCfg *configs.P2PConfig
+		m      *p2pModule
+	}
+	tests := []struct {
+		name               string
+		args               args
+		wantBootstrapNodes []string
+		wantErr            bool
+	}{
+		{
+			name: "unset boostrap nodes should yield no error and return DefaultP2PBootstrapNodes",
+			args: args{
+				p2pCfg: &configs.P2PConfig{},
+				m:      new(p2pModule),
+			},
+			wantErr:            false,
+			wantBootstrapNodes: defaultBootstrapNodes,
+		},
+		{
+			name: "empty string boostrap nodes should yield no error and return DefaultP2PBootstrapNodes",
+			args: args{
+				p2pCfg: &configs.P2PConfig{
+					BootstrapNodesCsv: "",
+				},
+				m: new(p2pModule),
+			},
+			wantErr:            false,
+			wantBootstrapNodes: defaultBootstrapNodes,
+		},
+		{
+			name: "untrimmed empty string boostrap nodes should yield no error and return DefaultP2PBootstrapNodes",
+			args: args{
+				p2pCfg: &configs.P2PConfig{
+					BootstrapNodesCsv: "     ",
+				},
+				m: new(p2pModule),
+			},
+			wantErr:            false,
+			wantBootstrapNodes: defaultBootstrapNodes,
+		},
+		{
+			name: "untrimmed string boostrap nodes should yield no error and return DefaultP2PBootstrapNodes",
```

Comment:
this does not return the default boostrap nodes

Reply:
bad copy-🍝 that helped finding a bug. Now fixed